src/machine.rs

```diff
@@ -296,7 +299,10 @@ impl<'mir, 'tcx> Machine<'mir, 'tcx> for Evaluator<'tcx> {
         id: AllocId,
         alloc: Cow<'b, Allocation>,
         kind: Option<MemoryKind<Self::MemoryKinds>>,
-    ) -> (Cow<'b, Allocation<Self::PointerTag, Self::AllocExtra>>, Self::PointerTag) {
+    ) -> (
+        Cow<'b, Allocation<Self::PointerTag, Self::AllocExtra>>,
+        Self::PointerTag,
+    ) {
```

Comment:
This got worse.

Reply:
I disagree, I like this (also no clue how to undo this without introducing type aliases and thus totally rewriting it so rustfmt doesn't touch it)

Reply:
The return type is now illegible where previously it fit on a line with a mostly reasonable width... how's that not worse?

Reply:
It's a tuple, and its fields are each on a line. Since tuple return types feel to me like the function has multiple return types, each of them being on its own line seems good to me.

That said, this is resolved by the new rustfmt.toml flag